easybuild/easyconfigs/g/gperftools/gperftools-2.5-foss-2016a.eb

```diff
@@ -0,0 +1,24 @@
+easyblock = 'ConfigureMake'
+
+name = "gperftools"
+version = "2.5"
+
+homepage = 'http://github.com/gperftools/gperftools'
+description = """gperftools are for use by developers so that they can create more robust applications.
+ Especially of use to those developing multi-threaded applications in C++ with templates.
+ Includes TCMalloc, heap-checker, heap-profiler and cpu-profiler."""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+sources = [SOURCE_TAR_GZ]
+source_urls = ['https://github.com/gperftools/gperftools/releases/download/%(namelower)s-%(version)s']
+
+builddependencies = [('Autotools', '20150215', '', ('GCCcore', '4.9.3'))]
```

Comment:
@FokkeDijkstra let's make this simply `('Autotools', '20150215')`, specifying the subtoolchain here is not needed?

This also relates to the likely switch of enabling `--minimal-toolchains` by default in the upcoming EasyBuild v3.0.0 (cfr. http://easybuild.readthedocs.io/en/latest/Manipulating_dependencies.html#using-minimal-toolchains-for-dependencies).